### PR TITLE
[FLINK-11331][table][docs] Fix errors in tableApi.md and functions.md

### PR DESCRIPTION
**Pull request by hequn8128**
## What is the purpose of the change

This pull request fixes errors in `tableApi.md` and `functions.md`.

## Brief change log

  - Fix example errors in tableApi.md.
  - Add the missing `quarters` and `weeks` udf in `functions.md` for java&scala

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/dev/table/functions.md [hunk 1]
```diff
@@ -3843,6 +3877,18 @@ NUMERIC.months
...
 
+    <tr>
+      <td>
+        {% highlight scala %}
+NUMERIC.week
+NUMERIC.weeks
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Creates an interval of milliseconds for <i>NUMERIC</i> weeks.</p>
```
[the remark this answers is not in the document]
Comment (hequn8128) [reply docs/dev/table/functions.md]: Good point!

**Comment by sunjincheng121** [comment]
Thanks for the update! @hequn8128 
will be merged.

**Comment by hequn8128** [comment]
@sunjincheng121 Thanks for the nice suggestion. 
I create another [issue](https://issues.apache.org/jira/browse/FLINK-11342) to address the example problem of built-in udfs. I think it would be very helpful if we have example for every function. Looking forward to have your suggestions.

Best, Hequn

**Comment by sunjincheng121** [comment]
Merging...

**Pull request closed without merging**